packages/server/src/profile/profile.controller.ts

```diff
@@ -62,6 +62,10 @@ export class ProfileController {
     private mailService: MailService,
     private organizationService: OrganizationService,
   ) {}
+
+  // Number of courses to retrieve from database per user
+  private SEARCH_COURSE_LIMIT = 10;
```

Comment:
Why have a limit?

Reply:
I will remove this as it's no longer used, but they idea is to limit # of data we pull from the database; because we don't have many tables indexed, this could impact performance in the future.